keps/sig-node/NNNN-memory-manager-hugepages-verification/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Memory Manager Hugepages Availability Verification
+kep-number: NNNN
+authors:
+  - "@srikalyan"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2024-12-24
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-node/1769-memory-manager"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.33"
```

Comment:
likewise, these will need to shifted using 1.36 as base

Reply:
Agreed. I'll shift all milestones using v1.36 as base:
- alpha: v1.36
- beta: v1.37  
- stable: v1.38